OWNERS

```diff
@@ -7,13 +7,15 @@ reviewers:
   - nikhita
   - palnabarun
   - Priyankasaggu11929
+  - mfahlandt
 approvers:
   - cblecker
   - MadhavJivrajani
   - mrbobbytables
   - nikhita
   - palnabarun
   - Priyankasaggu11929
+  - mfahlandt
```

Comment:
Drop the addition here. This is only for @kubernetes/owners and not ContribEx leads.